kano_feedback/DataSender.py

```diff
@@ -372,6 +372,43 @@ def get_hdmi_info():
     return res + edid
 
 
+def get_screen_log():
+    """Get display information using kano-settings display functions.
+
+    Returns:
+        dict: An aggregate of display characteristics
+    """
+
+    try:
+        import json
+
+        from kano_settings.system.display import get_edid, get_edid_name, get_status, \
+            list_supported_modes, get_optimal_resolution_mode, override_models
+
+        edid = get_edid()
+        model = get_edid_name(use_cached=False)
+
+        override_models(edid, model)
+
+        status = get_status()
+        supported = list_supported_modes(use_cached=False)
+        optimal = get_optimal_resolution_mode(edid, supported)
+
+        log_data = {
+            'model': model,
+            'status': status,
+            'edid': edid,
+            'supported': supported,
+            'optimal': optimal,
+        }
+        log = json.dumps(log_data, sort_keys=True, indent=4)
+    except:
+        import traceback
```

Comment:
Only a suggestion, importing stuff in an exception looks a bit risky to me.

Reply:
Good point! It was merely to avoid importing a bunch of things unless required. Fixed 👍

Reply:
That's a good point. You may want to `try` to import inside the `exception` (although maybe nesting multiple exceptions isn't what you wanted)!

```
except:
    try:
        import traceback
    except ImportError:
         foo
```